pyproject.toml

```diff
@@ -30,9 +30,7 @@ filterwarnings = [
     # This is just to prevent pytest from exiting if pytest-xdist isn't installed.
     "ignore:Unknown config option.*looponfailroots:pytest.PytestConfigWarning",
 
-    # TODO(joshuarli): Enable this and address all after Django 2.0 lands.
-    "ignore::django.utils.deprecation.RemovedInDjango21Warning",
```

Comment:
Opting to not ignore RemovedInDjango22Warning so other people can fix them if they introduce it (which will fail pytest).